<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>MFR: Multi-Feature Relatedness modeln</title>

	<link rel="stylesheet" media="screen" href="./page/download.css?v=8may2013">
	<link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="author" content="Dave Shea">
	<meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
	<meta name="robots" content="all">


	<!--[if lt IE 9]>
	<script src="script/html5shiv.js"></script>
	<![endif]-->
</head>

<!--



	View source is a feature, not a bug. Thanks for your curiosity and
	interest in participating!

	Here are the submission guidelines for the new and improved csszengarden.com:

	- CSS3? Of course! Prefix for ALL browsers where necessary.
	- go responsive; test your layout at multiple screen sizes.
	- your browser testing baseline: IE9+, recent Chrome/Firefox/Safari, and iOS/Android
	- Graceful degradation is acceptable, and in fact highly encouraged.
	- use classes for styling. Don't use ids.
	- web fonts are cool, just make sure you have a license to share the files. Hosted 
	  services that are applied via the CSS file (ie. Google Fonts) will work fine, but
	  most that require custom HTML won't. TypeKit is supported, see the readme on this
	  page for usage instructions: https://github.com/mezzoblue/csszengarden.com/

	And a few tips on building your CSS file:

	- use :first-child, :last-child and :nth-child to get at non-classed elements
	- use ::before and ::after to create pseudo-elements for extra styling
	- use multiple background images to apply as many as you need to any element
	- use the Kellum Method for image replacement, if still needed. http://goo.gl/GXxdI
	- don't rely on the extra divs at the bottom. Use ::before and ::after instead.

		
-->

<body id="css-zen-garden">
<div class="page-wrapper">

	<section class="intro" id="zen-intro">
	<div class="hh">
		<iframe src="./header.html" frameborder="0" scrolling="no" width="750px" height="215px" ></iframe>
		</div>
		

		<div class="preamble" id="zen-preamble" role="article">
		  <h1>Functions</h1>
		  <p>&nbsp;&nbsp;We have saved the trained and developed <abbr title="Multi-Feasures Relatedness">MFR</abbr> model as an R data, which can be freely downloaded and modified for academic uses. We also provided R function script files for computing gene-pair features and <abbr title="Multi-Feasures Relatedness">MFR</abbr> values, and a guidance file, an example R data file with the result file and a gene expression R data file for teaching and showing others how to use these resource and tools.</p>
		  <ul class="can">
  				<li>
  				  <p><a href= "data/function files/mfr.model.zip">MFR model.RData (9.1M)</a></p></li>
  				<li>
  				  <p><a href= "data/function files/featureComputing.R">featureComputing.R (3.9K)</a></p></li>
  				  <li>
  				  <p><a href= "data/function files/MFRcomputing.R">MFRcomputing.R (334Byte)</a></p></li>
  				<li>
  				  <p><a href= "data/function files/exp.matrix.uniprot.human.GPL570.602.zip">exp.matrix.uniprot.human.GPL570.602.RData (1.5G)</a></p></li>
  				  <li>
  				  <p><a href= "data/function files/example.gene.pair.zip">example.gene.pair.RData (432Byte)</a></p></li>
  				<li>
  				  <p><a href= "data/function files/example.result.zip">example.result.CSV (2.1K)</a></p></li>
  				  <li>
  				  <p><a href= "data/function files/guidance.docx">guidance.docx (41K)</a></p></li>
			</ul>
		</div>
   	    <div class="preamble2" id="zen-preamble2" role="article">
    	 <h1>Dependent Files</h1>
		  <p>&nbsp;&nbsp;Here are the dependent files for computing gene-pair features and <abbr title="Multi-Feasures Relatedness">MFR</abbr> values.</p>

		  <ul class="can">
  				<li>
  				  <p><a href= "data/dependent files/gene.pair.htridb.zip">gene.pair.htridb.RData (200K)</a></p></li>
  				<li>
  				  <p><a href= "data/dependent files/geneSpeciesmapping.zip">geneSpeciesmapping.RData (171K)</a></p></li>
  				  <li>
  				  <p><a href= "data/dependent files/goTerm2Fathers2NumOfSonMappingData.zip">goTerm2Fathers2NumOfSonMappingData.RData (751K)</a></p></li>
  				<li>
  				  <p><a href= "data/dependent files/goTermCFatherSonMappingData.zip">goTermCFatherSonMappingData.RData (4.2K)</a></p></li>
  				  <li>
  				  <p><a href= "data/dependent files/hsa.gene.pair.reactome2.zip">hsa.gene.pair.reactome2.RData (422K)</a></p></li>
  				<li>
  				  <p><a href= "data/dependent files/human_goAnnotaion.zip">human_goAnnotaion.RData (4.1M)</a></p></li>
  				  <li>
  				  <p><a href= "data/dependent files/pcc.m.all2.zip">pcc.m.all2.RData (1.2G)</a></p></li>
			</ul>
     
      </div>
     <div class="preamble1" id="zen-preamble1" role="article">
       <h1>Resource</h1>
       <p>&nbsp;&nbsp;<abbr title="Multi-Feasures Relatedness">MFR</abbr> model was trained over the gene-pair samples collected from the <a href="http://coxpresdb.jp">COXPRESdb</a>, <a href="http://www.kegg.jp">KEGG</a> and <a href="http://www.grnpedia.org/trrust/">TRRUST</a> databases and the dataset of the <a href="https://pubs.acs.org/doi/abs/10.1021/pr100618t"> study of Pan at el.</a>, and, in which the positive gene pairs are the coexpressed gene pairs, gene pairs with PPIs, involved in same pathways, or with transcriptional regulatory relationships, and the negatives are the opposites. We have also used <abbr title="Multi-Feasures Relatedness">MFR</abbr> model to measure relatedness of the gene pairs in the  geneFriends, DIP, cancer and KEGG-metabolism datasets, in which the positive gene pairs in the <a href="http://www.genefriends.org">geneFriends</a> dataset are the gene pairs with co-expressions collected from the geneFriends  database and the negatives are gene pairs by permutation of the first genes with the second genes in the positive gene pairs; the positive gene pairs in the <a href="http://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a> dataset are gene pairs with PPIs collected from the <abbr title="Database of Interacting Proteins">DIP</abbr> database and the negatives are by permutation of the first genes with the second genes in the positive gene pairs; the gene pair in the <a href="https://cancergenome.nih.gov">TCGA</a> cancer datasets are composited by the metabolic genes up-regulated in four cancer types including <abbr title="bladder urothelial carcinoma">BLCA</abbr>, <abbr title="breast invasive carcinoma">BRCA</abbr>, <abbr title="colon adenocarcinoma">COAD</abbr> and <abbr title="lung adenocarcinoma">LUAD</abbr>, and the gene pairs in KEGG-metabolism dataset are composited by the genes involved in metabolism pathways collected from the <a href="http://www.kegg.jp">KEGG</a> database. All the datasets and the results mentioned above are downloaded freely for further verification and biological analysis.</p>
       <ul class="can">
            <li>
			  <p><a href= "data/resource data/tr_dv_ts.dataset.zip">MFR model dataset (11M).</a></p>
			</li>
			<li>
			  <p><a href= "data/resource data/GeneFriends.dataset.zip">geneFriends dataset (1.4M).</a></p>
			</li>
  			<li>
  			  <p><a href= "data/resource data/DIP.dataset.zip">DIP dataset (235K).</a></p>
  			</li>
  			<li>
			  <p><a href= "data/resource data/blca.dataset.zip">BLCA dataset (1.9M).</a></p>
			</li>
  			<li>
  			  <p><a href= "data/resource data/brca.dataset.zip">BRCA dataset (961K).</a></p>
  			</li>
  			<li>
			  <p><a href= "data/resource data/coad.dataset.zip">COAD dataset (1.3M).</a></p>
			</li>
  			<li>
  			  <p><a href= "data/resource data/luad.dataset.zip">LUAD dataset (1.9M).</a></p>
  			</li>
  			<li>
  			  <p><a href= "data/resource data/keggMetabolism.dataset.zip">KEGG-metabolism dataset (60M).</a></p>
  			</li>
			</ul>
           
      </div>
	</section>
<aside class="sidebar" role="complementary"> </aside>
		

		
		</div>
<div class= "fh" id="zen-fh" role="foot"><iframe src="foot.html" frameborder="0" scrolling="no" width="750px" height="35px" ></iframe>
</div>

<!--

	These superfluous divs/spans were originally provided as catch-alls to add extra imagery.
	These days we have full ::before and ::after support, favour using those instead.
	These only remain for historical design compatibility. They might go away one day.
		
-->
<div class="extra1" role="presentation"></div><div class="extra2" role="presentation"></div><div class="extra3" role="presentation"></div>
<div class="extra4" role="presentation"></div><div class="extra5" role="presentation"></div><div class="extra6" role="presentation"></div>

</body>
</html>