<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>MFR: Multi-Feature Relatedness modeln</title>

	<link rel="stylesheet" media="screen" href="./download.css?v=8may2013">
	<link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="author" content="Dave Shea">
	<meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
	<meta name="robots" content="all">


	<!--[if lt IE 9]>
	<script src="script/html5shiv.js"></script>
	<![endif]-->
</head>

<!--



	View source is a feature, not a bug. Thanks for your curiosity and
	interest in participating!

	Here are the submission guidelines for the new and improved csszengarden.com:

	- CSS3? Of course! Prefix for ALL browsers where necessary.
	- go responsive; test your layout at multiple screen sizes.
	- your browser testing baseline: IE9+, recent Chrome/Firefox/Safari, and iOS/Android
	- Graceful degradation is acceptable, and in fact highly encouraged.
	- use classes for styling. Don't use ids.
	- web fonts are cool, just make sure you have a license to share the files. Hosted 
	  services that are applied via the CSS file (ie. Google Fonts) will work fine, but
	  most that require custom HTML won't. TypeKit is supported, see the readme on this
	  page for usage instructions: https://github.com/mezzoblue/csszengarden.com/

	And a few tips on building your CSS file:

	- use :first-child, :last-child and :nth-child to get at non-classed elements
	- use ::before and ::after to create pseudo-elements for extra styling
	- use multiple background images to apply as many as you need to any element
	- use the Kellum Method for image replacement, if still needed. http://goo.gl/GXxdI
	- don't rely on the extra divs at the bottom. Use ::before and ::after instead.

		
-->

<body id="css-zen-garden">
<div class="page-wrapper">

	<section class="intro" id="zen-intro">
			<div class="hh">
					<header role="banner">
					<h4> a </h4>
						<h1>GeneQC</h1>
						<h2>Gene expression level Quality Check</h2>
					</header>
					<div class = "dh">
						<ul>
							<li><a class = "active" href="./home.html">Home</a></li>
											<li><a href="./author.html">Authors</a></li>
											<li><a href="./download.html">Download</a></li>
							<li><a href="./tutorial.html">Tutorial</a></li>
							<li><a href="./program.html">Program</a></li>
						</ul>
					
					</div>
					</div>
		

		<div class="preamble" id="zen-preamble" role="article">
		  <h1>Download</h1>
		  <p>We have saved the trained <abbr title="Multi-Feasures Relatedness">MFR</abbr> model as a R data, which can be freely downloaded and modified for academic uses.</p>
		  <ul class="can">
  				<li>
  				  <p><a href="../data/model.mfr.RData">The trained MFR model.</a></p></li>
  				<li>
  				  <p><a href="../data/examples.mfr.zip">The example files.</a></p></li>
			</ul>
     
      </div>
     <div class="preamble1" id="zen-preamble1" role="article">
		  <h1>Recourse</h1>
		  <p>We have used MFR model to measure  relatedness of the gene pairs in the <abbr title="Database of Interacting Proteins">DIP</abbr>, geneFriends, cancer and  <abbr title="Kyoto Encyclopedia of Genes and Genomes">KEGG</abbr>-metabolism datasets, in which the positive samples in the <abbr title="Database of Interacting Proteins">DIP</abbr> dataset are  gene pairs with <abbr title="Proteinâ€“Protein Interaction">PPI</abbr>s collected from the <a href="http://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a> database and the negative samples  are gene pairs composited randomly by the genes in the positive samples, the  positive samples in the geneFriends dataset are the gene pairs with  co-expressions collected from the <a href="http://www.genefriends.org">geneFriends</a> database and the negative samples are  gene pairs randomly by the genes in the positive samples, the samples in the <a href="https://cancergenome.nih.gov">TCGA</a> cancer datasets are gene pairs composited by the genes up-regulated in four cancer  types including <abbr title="bladder urothelial carcinoma">BLCA</abbr>, <abbr title="breast invasive carcinoma">BRCA</abbr>, <abbr title="colon adenocarcinoma">COAD</abbr> and <abbr title="lung adenocarcinoma">LUAD</abbr>, and the samples in KEGG-metabolism dataset are the gene pairs composited by the genes involved in metabolism pathways collected from the <a href="http://www.kegg.jp">KEGG</a> database. All the results mentioned above are downloaded freely  for further  verification and biological analysis. </p>
         <ul class="can">
			<li>
			  <p><a href="../data/dip.mfr.csv">The DIP dataset.</a></p>
			</li>
  			<li>
  			  <p><a href="../data/geneFriends.mfr.csv">The geneFriends dataset.</a></p>
  			</li>
  			<li>
			  <p><a href="../data/blca.mfr.csv">The BLCA dataset.</a></p>
			</li>
  			<li>
  			  <p><a href="../data/brca.mfr.csv">The BRCA dataset.</a></p>
  			</li>
  			<li>
			  <p><a href="../data/coad.mfr.csv">The COAD dataset.</a></p>
			</li>
  			<li>
  			  <p><a href="../data/luad.mfr.csv">The LUAD dataset.</a></p>
  			</li>
  			<li>
  			  <p><a href="../data/keggMetabolism.mfr.csv">The KEGG-metabolism dataset.</a></p>
  			</li>
			</ul>
           
      </div>
	</section>
<aside class="sidebar" role="complementary"> </aside>


</div>

<!--

	These superfluous divs/spans were originally provided as catch-alls to add extra imagery.
	These days we have full ::before and ::after support, favour using those instead.
	These only remain for historical design compatibility. They might go away one day.
		
-->
<div class="extra1" role="presentation"></div><div class="extra2" role="presentation"></div><div class="extra3" role="presentation"></div>
<div class="extra4" role="presentation"></div><div class="extra5" role="presentation"></div><div class="extra6" role="presentation"></div>

</body>
</html>