<!DOCTYPE html>
<html lang="en">
		<script src="/js/particles.min.js"></script>

<head>
	<meta charset="utf-8">
	<title>GeneQC: Gene expression level Quality Check</title>

	<link rel="stylesheet" media="screen" href="./home.css?v=8may2013">
	<link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="author" content="Cankun Wang">
	<meta name="description" content="Gene expression level Quality Check">
	<meta name="robots" content="all">


	<!--[if lt IE 9]>
	<script src="script/html5shiv.js"></script>
	<![endif]-->
</head>


<body id="css-zen-garden">
<div class="page-wrapper">

	<section class="intro" id="zen-intro">
	<div class="hh">
		<header role="banner">
		<h4> a </h4>
			<h1>GeneQC</h1>
			<h2>Gene expression level Quality Check</h2>
		</header>
		<div class = "dh">
			<ul>
				<li><a class = "active" href="./home.html">Home</a></li>
                <li><a href="./author.html">Authors</a></li>
                <li><a href="./download.html">Download</a></li>
				<li><a href="./tutorial.html">Tutorial</a></li>
				<li><a href="./program.html">Program</a></li>
			</ul>
		
		</div>
		</div>
		

		<div class="preamble" id="zen-preamble" role="article">
		  <h1>Why is GeneQC needed?</h1>
		  <p>Mapping Uncertainty, in which a given RNA-seq read can be mapped to multiple genomic locations, is a prominent issue in reference-based RNA-seq analyses throughout most eukaryotic species.  This issue can result in biased gene expression estimations which affects all downstream analyses, from assembly and differential gene expression to regulatory network elucidation. </p>
		  <div class="fig1"><img border="0" src="./img/figure1.png" alt="GeneQC figure 1.2" onclick="window.open('./img/figure1.png', '_blank');" >
		  <p class="fig">Figure 1. To be added</p></div>
	  </div>

		<div class="preamble" id="zen-preamble" role="article">
			<h1>How do we approach this problem?</h1>
			<p>The first step to addressing mapping uncertainty is to develop a method for determining how severe the problem is for a given dataset. In GeneQC, we extract usable features from two genetic levels, genomic and transcriptomic. From the genomic level, we look at the sequence similarity for two genomic locations, whereas for the transcriptomic level, we investigate the proportion of shared reads for two genes or transcripts after initial alignment.</p>
			<div class="fig1"><img border="0" src="./img/figure2.jpg" alt="GeneQC " onclick="window.open('./img/figure2.jpg', '_blank');" >
			<p class="fig">Figure 2. To be added</p></div>
		</div>


		<div class="preamble" id="zen-preamble" role="article">
			<h1>Our Process!</h1>
			<p>These two levels of information are combined using advanced mathematical and statistical approaches to develop a distinct score, referred to as the D-score, to provide a clear measure of the level of mapping uncertainty for each annotated gene for a particular species. Additionally, mixture model distributions are used to determine categorizations for the mapping uncertainty, and thus recommendations for which genetic expression estimates are reliable following the read alignment step. </p>
			<div class="fig1"><img border="0" src="./img/" alt="GeneQC " onclick="window.open('./img/figure1.2.jpg', '_blank');" >
			<p class="fig">To be added</p></div>
		</div>

		<div class="preamble1" id="zen-preamble1" role="article">
		  <h1>Citing GeneQC</h1>
		  <p>To be added</p>
		  <p></p>
		  <p></p>
	  </div>
	  <div id="copyright">
		<p >
			<strong>Copyright 2018 Â© <a href="http://prod.sdstate.edu/agronomy-horticulture-and-plant-science/bioinformatics-and-mathematical-biosciences-lab" target="_blank">BMBL</a>, <a href="http://prod.sdstate.edu/" target="_blank">SDSU</a>. All rights reserved. <br></strong><a href="mailto:qin.ma@sdstate.edu" title="qin.ma@sdstate.edu">Contact us: qin.ma@sdstate.edu</a>  
		</br>
		</p>
	</div>
	</section>
<aside class="sidebar" role="complementary"> </aside>


</div>

<!--

	These superfluous divs/spans were originally provided as catch-alls to add extra imagery.
	These days we have full ::before and ::after support, favour using those instead.
	These only remain for historical design compatibility. They might go away one day.
		
-->
<div class="extra1" role="presentation"></div><div class="extra2" role="presentation"></div><div class="extra3" role="presentation"></div>
<div class="extra4" role="presentation"></div><div class="extra5" role="presentation"></div><div class="extra6" role="presentation"></div>

</body>
</html>